code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -49,12 +49,12 @@
 		if(pain_shock_stage == 150)
```

Comment:
is this check even necessary? `if(pain_shock_stage >= 150)` should cover it

Reply:
The entire thing happens if the pain level increases, and it does 1 by 1. When it hits 150 it does a guaranteed knockdown.